sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ### Breaking Changes
 - `NotificationHook.ExternalLink` and `WebNotificationHook.Endpoint` are now of type `Uri`.
+- Removed getter from `GetIncidentsForDetectionConfigurationOptions.DimensionsToFilter`. Elements can be added directly to it without user instantiation.
```

Comment:
setter, no?

Reply:
You're right. I'll merge this one and fix the changelog in my next PR.